packages/@lwc/engine-server/src/__tests__/fixtures/dynamic-components/modules/x/dynamic/dynamic.js

```diff
@@ -0,0 +1,6 @@
+import { LightningElement, track } from 'lwc';
+import Test from 'x/test';
+
+export default class DynamicCtor extends LightningElement {
+    @track customCtor = Test;
```

Comment:
Should we also have a server test for when the `ctor` is undefined?

Reply:
Added a test in the latest [commit](https://github.com/salesforce/lwc/blob/1658d36d1a001bfce3a4a441323e9d4caf473f48/packages/%40lwc/engine-server/src/__tests__/fixtures/dynamic-component-no-ctor/index.js)!